apps/explorer/src/ui/CoinsStack.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Sui, Unstaked } from '@mysten/icons';
+import clsx from 'clsx';
+
+interface CoinIconProps {
+    type: string;
+}
+
+function getRenderCoin(type: string) {
```

Comment:
Why not just fetch the coin metadata and actually display the coin logos?

Reply:
Where do I find the meta data?